specification-discussions/20200214-subgraph-gen1-facilitator.md

```diff
@@ -21,7 +21,11 @@ Subgraph will be responsible for indexing and querying data from blockchains. It
 1. As a user of subgraph, I should be able to deploy subgraph for auxiliary.
 
 ## Tasks
-1. Create new repoository mosaic-subgraph
+1. Create new repository mosaic-subgraph.
+
+1. While deploying subgraph, unique subgraph name needs to be provided for deployment. The standard format followed is
```

Comment:
this is not unique. the contract type name should imply location on origin or metachain. see conversation for alternative naming suggestion

Reply:
Updated the proposal.